app/src/Components/Project/ProjectRegisterInterest.tsx

```diff
@@ -4,6 +4,7 @@ import { ScrollView } from 'react-native-gesture-handler'
 import styled from 'styled-components/native'
 import dayjs from 'dayjs'
 import ProjectHeading from './ProjectHeading'
+import DateControls from '../Forms/DateControls'
```

Comment:
I get an error saying this module can't be found -- maybe it's pointing to a component that got moved/renamed?

Reply:
It's not called so I've removed the import.